src/components/Sidebar/sidebar.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import clsx from 'clsx';
+import { makeStyles } from '@material-ui/core/styles';
+import Drawer from '@material-ui/core/Drawer';
+import Button from '@material-ui/core/Button';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import InboxIcon from '@material-ui/icons/MoveToInbox';
+import MailIcon from '@material-ui/icons/Mail';
+import styled from 'styled-components';
+
+const StyledDrawer = styled(Drawer)`
```

Comment:
Remove styled components
Create a css file
follow the filename guiddeline
Sidebar.component.jsx
^ capital first letter